suites/nautilus/upgrades/upgrade_4.x-1_to_4.x.yaml

```diff
@@ -0,0 +1,100 @@
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+- test:
+    name: ceph ansible install rhcs 4.x-1 from cdn
+    polarion-id: CEPH-83571467
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4 
+        ceph_stable_release: nautilus
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_docker_registry_auth: false
+        dashboard_enabled: false
+        copy_admin_key: True
+        ceph_conf_overrides:
+          global:
+            osd_pool_default_pg_num: 128
+            osd_default_pool_size: 2
+            osd_pool_default_pgp_num: 128
+            mon_max_pg_per_osd: 4096
+            mon_allow_pool_delete: True
+          client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: test cluster 4.x-1 cdn setup using ceph-ansible
+    abort-on-fail: True
+- test:
+    name: Upgrade ceph ansible to 4.x
+    polarion-id: CEPH-11110
+    module: test_ansible_upgrade.py
+    config:
+      ansi_config:
+        ceph_origin: distro
+        ceph_stable_release: nautilus
+        ceph_repository: rhcs
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_stable: True
+        ceph_stable_rh_storage: True
+        fetch_directory: ~/fetch
+        copy_admin_key: true
+        dashboard_enabled: False
+        upgrade_ceph_packages: True
+        ceph_rhcs_version: 4
+        ceph_iscsi_config_dev: false
+        node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.1
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-3-dashboard-rhel7:3
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.1
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:4.1
+        ceph_conf_overrides:
+          global:
+            osd_pool_default_pg_num: 64
+            osd_default_pool_size: 2
+            osd_pool_default_pgp_num: 64
+            mon_max_pg_per_osd: 1024
+            mon_allow_pool_delete: true
+            client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: Test Ceph-Ansible rolling update -4.x-1 -> 4.x
```

Comment:
same here , remove x-1 convention and make it something like "4.x cdn to 4.x latest internal build"

Reply:
Done